README.rst

```diff
@@ -0,0 +1,164 @@
+PennyLane-Qulacs Plugin
+######################
+
+.. image:: https://github.com/XanaduAI/pennylane-cirq/blob/master/doc/_static/puzzle_qulacs.png?raw=true
+    :align: center
+    :width: 200px
+    :target: javascript:void(0);
+
+|
+
+.. header-start-inclusion-marker-do-not-remove
+
+The PennyLane-Qulacs plugin integrates the Qulacs quantum computing framework with PennyLane's
+quantum machine learning capabilities.
+
+`PennyLane <https://pennylane.readthedocs.io>`__ is a cross-platform Python library for quantum machine
+learning, automatic differentiation, and optimization of hybrid quantum-classical computations.
+
+`Qulacs <https://github.com/qulacs>`__ is a software library for quantum computing, written in C++ and with GPU support.
+
+.. header-end-inclusion-marker-do-not-remove
+
+The plugin documentation can be found here: `<https://pennylane-qulacs.readthedocs.io/en/latest/>`__.
+
+Features
+========
+
+* Provides access to Qulacs' simulator backend via the ``qulacs.simulator`` device
+
+* Support for all PennyLane core functionality
+
+.. benchmarks-start-inclusion-marker-do-not-remove
+
+Benchmarks
+==========
+
+We ran a 100 executions of 4 layer quantum neural 
+network `strongly entangling layer <https://pennylane.readthedocs.io/en/latest/code/api/pennylane.templates.layers.StronglyEntanglingLayer.html>`_
+and compared the runtimes between CPU and GPU.
+
+.. image:: https://raw.githubusercontent.com/soudy/pennylane-qulacs/master/images/qnn_cpu_vs_gpu.png
+    :align: center
+    :width: 60%
+    :target: javascript:void(0);
+
+|
+
+.. image:: https://raw.githubusercontent.com/soudy/pennylane-qulacs/master/images/qulacs_table.png
+    :align: center
+    :width: 60%
+    :target: javascript:void(0);
+
+|
+
+
+.. benchmarks-end-inclusion-marker-do-not-remove
+
+
+.. installation-start-inclusion-marker-do-not-remove
+
+Installation
+============
+
+This plugin requires Python version 3.6 or above, as well as PennyLane
+and Qulacs. Installation of this plugin, as well as all dependencies, can be done using ``pip``:
+
+.. code-block:: bash
+
+    $ pip install pennylane-qulacs
+
+Alternatively, you can install PennyLane-Qulacs from the `source code <https://github.com/XanaduAI/pennylane-qulacs>`__
+by navigating to the top directory and running:
+
+.. code-block:: bash
+
+	$ python setup.py install
+
+Dependencies
+~~~~~~~~~~~~
+
+PennyLane-Qulacs requires the following libraries be installed:
+
+* `Python <http://python.org/>`__ >= 3.6
+
+as well as the following Python packages:
+
+* `PennyLane <http://pennylane.readthedocs.io/>`__ >= 0.9
+* `Qulacs <https://docs.qulacs.org/en/latest/>`__ 
```

Comment:
Couldn't find the current version?

Reply:
The [latest version on PyPI](https://pypi.org/project/Qulacs/) is 0.1.9.

Reply:
Great, will add